pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -152,6 +170,17 @@ private static void SetScanType(ChromSource source, double x, double y)
             CheckFullScanSelection(x, y);
         }
 
+        private static void SetShowAnnotations(bool isChecked)
+        {
+            RunUI(() => SkylineWindow.GraphFullScan.SetShowAnnotations(isChecked));
+        }
+
+        private static void TestAnnotations(string[] annotationText)
+        {
+                var graphLabels = SkylineWindow.GraphFullScan.ZedGraphControl.GraphPane.GraphObjList.OfType<TextObj>()
+                    .Select(label => label.Text).ToList();

```

Comment:
Since you're going to be using "Contains" later on, it would be a little faster to do "ToHashSet()" here instead of "ToList()".

Reply:
Fixed.